cmd/apply-crds/main.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright (c), NVIDIA CORPORATION.  All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	"github.com/NVIDIA/k8s-operator-libs/pkg/crdutil"
+)
+
+var logger = log.New()
+
+type config struct {
+	Debug     bool
+	crdsPaths []string
+}
+
+func main() {
+	config := config{}
+
+	// Create the top-level CLI
+	c := cli.NewApp()
+	c.Name = "apply-crds"
+	c.Usage = "Tools for managing Custom Resource Definitions (CRDs) for NVIDIA GPU Operator"
+	c.Version = "0.1.0"
```

Comment:
Should we set the version in the same way that we set the other CLI versions. This *should* give us the git commit SHA as well, which could e useful for debugging.